DataTrackExample/ViewController.swift

```diff
@@ -0,0 +1,468 @@
+    // deserialized using this convention.
+    let kTouchBegan = "mouseDown";
+    let kTouchPoint = "mouseCoordinates";
+    let kXCoordinate = "x";
+    let kYCoordinate = "y";
+    
+    // Video SDK components
+    var room: TVIRoom?
+    var localDataTrack: TVILocalDataTrack!
+    var localParticipant: TVILocalParticipant?
+    var remoteView: TVIVideoView?
+    
+    // Private members
+    var drawers = Dictionary<TVIDataTrack, Drawer>()
+    
+    // By default app sends `Data`. Flip this flag to send json String instead.
+    let useSendStringAPI = false
+    
+    // MARK: UI Element Outlets and handles
+    
+    @IBOutlet weak var connectButton: UIButton!
+    @IBOutlet weak var disconnectButton: UIButton!
+    @IBOutlet weak var messageLabel: UILabel!
+    @IBOutlet weak var roomTextField: UITextField!
+    @IBOutlet weak var roomLine: UIView!
+    @IBOutlet weak var roomLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.disconnectButton.isHidden = true
+        self.roomTextField.autocapitalizationType = .none
+        self.roomTextField.delegate = self
+    }
+
+    // MARK: IBActions
+    @IBAction func connect(sender: AnyObject) {
+        // Configure access token either from server or manually.
+        // If the default wasn't changed, try fetching from server.
+        if (accessToken == "TWILIO_ACCESS_TOKEN") {
+            do {
+                accessToken = try TokenUtils.fetchToken(url: tokenUrl)
+            } catch {
+                let message = "Failed to fetch access token"
+                logMessage(messageText: message)
+                return
+            }
+        }
+
+        let dataTrackOptions = TVIDataTrackOptions.init { (builder) in
+            builder.isOrdered = true
+        }
+        
+        self.localDataTrack = TVILocalDataTrack.init(options: dataTrackOptions, name: "Draw")
+        
+        // Preparing the connect options with the access token that we fetched (or hardcoded).
+        let connectOptions = TVIConnectOptions.init(token: accessToken) { (builder) in
+            
+            builder.dataTracks = [self.localDataTrack]
+
+            // The name of the Room where the Client will attempt to connect to. Please note that if you pass an empty
+            // Room `name`, the Client will create one for you. You can get the name or sid from any connected Room.
+            builder.roomName = self.roomTextField.text
+        }
+
+        // Connect to the Room using the options we provided.
+        room = TwilioVideo.connect(with: connectOptions, delegate: self)
+
+        logMessage(messageText: "Attempting to connect to room \(String(describing: self.roomTextField.text))")
+
+        self.showRoomUI(inRoom: true)
+        self.dismissKeyboard()
+    }
+    
+    @IBAction func disconnect(sender: AnyObject) {
+        self.room!.disconnect()
```

Comment:
Use `if let` binding on self.room instead of force unwrapping.

```
if let room = self.room {
    room.disconnect()
    logMessage(messageText: "Attempting to disconnect from room \(room.name)")
}
```